src/Core/CommandBus/TacticianCommandBusAdapter.php

```diff
@@ -27,30 +27,57 @@
 namespace PrestaShop\PrestaShop\Core\CommandBus;
 
 use League\Tactician\CommandBus;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Serializer\SerializerInterface;
 
 /**
  * Class TacticianCommandBusAdapter is Tactician's CommandsBus implementation for PrestaShop's contract.
  */
 final class TacticianCommandBusAdapter implements CommandBusInterface
 {
+    const LOG_MESSAGE_FORMAT = 'CQRS handle %s: %s';
+
     /**
      * @var CommandBus
      */
     private $bus;
 
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
     /**
      * @param CommandBus $bus
+     * @param LoggerInterface $logger
+     * @param SerializerInterface $serializer
      */
-    public function __construct(CommandBus $bus)
-    {
+    public function __construct(
+        CommandBus $bus,
+        LoggerInterface $logger,
+        SerializerInterface $serializer
+    ) {
         $this->bus = $bus;
+        $this->logger = $logger;
+        $this->serializer = $serializer;
     }
 
     /**
      * {@inheritdoc}
      */
     public function handle($command)
     {
+        $this->logger->info(sprintf(
+            self::LOG_MESSAGE_FORMAT,
+            get_class($command),
+            $this->serializer->serialize($command, 'json')
```

Comment:
This may be dangerous to log this since the command may contain some sensitive data